sidebars.js

```diff
@@ -217,6 +217,11 @@ const sidebars = {
           label: "Wallet With Celestia Node",
           id: "developers/celestia-node-key"
         },
+        {
+          type: "doc",
+          label: "Managing Wallets in Docker",
```

Comment:
Title doesn't say on whether this is for Celestia Node or something else

Reply:
awesome, fixed. thanks yaz

Reply:
although, `Managing Celestia Node Keys in Docker` is a little long for sidebar

Reply:
would it make sense to add docker as a subsection to `celestia-node-key` doc?

Reply:
Maybe? It seems like it may not be missed if it is it's own page, though

Reply:
yep good idea